src/cargo/ops/registry/publish.rs

```diff
@@ -418,7 +420,33 @@ fn transmit(
             .map(|(feat, values)| {
                 (
                     feat.to_string(),
-                    values.iter().map(|fv| fv.to_string()).collect(),
+                    values
```

Comment:
@epage , it sounds like it may be more better to strip the feature here??
https://github.com/rust-lang/cargo/blob/9e6288efe0bbccc53d95080070524e4640769339/src/cargo/util/toml/mod.rs#L939-L955

Reply:
That still requires stripping features twice
- Once for the Summary
- Once for `Cargo.toml`